cfgov/cfgov/settings/production.py

```diff
@@ -86,7 +86,7 @@
 
 STATIC_ROOT = os.environ['DJANGO_STATIC_ROOT']
 
-CACHES.update({
+CACHES = {
     'default': {
         'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache',
         'LOCATION': '/tmp/eregs_cache',
```

Comment:
![applause](http://cdn.theatlantic.com/static/mt/assets/science/theatermove.gif)

I think with this PR you're purging the last vestiges of eRegs!

Reply:
Sadly, this applause is premature. `/tmp/eregs_cache` lives on even after this PR, as the `default` Django cache. This PR just removes the `default_fragment_cache` Django cache.